migtests/tests/pg/datatypes/validate

```diff
@@ -26,6 +26,7 @@ EXPECTED_ROW_COUNT = {
 	'datetime_type2': 2, 
 	'null_and_default' :2,
 	'decimal_types': 3,
+	'hstore_example': 13,
```

Comment:
let's also add data validations (for each row), for source-target, source-source-replica, target-source-replica(ignore if this is not supported). 

You can use `get_distinct_values_of_column_of_table`